### PR TITLE
AV-188865 Secret Handling Fix

### PR DESCRIPTION
**Pull request by pkoshtavmware**
AV-188865 Secret Handling Fix

### CODE REVIEW

File: ako-gateway-api/nodes/dequeue_ingestion.go [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+		modelName := lib.GetModelName(lib.GetTenant(), akogatewayapilib.GetGatewayParentName(parentNs, parentName))
+
+		modelFound, modelIntf := objects.SharedAviGraphLister().Get(modelName)
+		if !modelFound || modelIntf == nil {
+			utils.AviLog.Errorf("key: %s, msg: no model found: %s", key, modelName)
+			continue
+		}
+		if objType == utils.Secret {
+			handleGateway(namespace, parentName, fullsync, lib.Gateway+"/"+gatewayNsName)
+			continue
```
Comment (swathinsankaran) [inline ako-gateway-api/nodes/dequeue_ingestion.go:67]: Where are we saving the model and publishing to rest layer in this case?

Comment (pkoshtavmware) [reply ako-gateway-api/nodes/dequeue_ingestion.go:67]: That code is already present in handleGateway function
load-balancer-and-ingress-services-for-kubernetes/ako-gateway-api/nodes/dequeue_ingestion.go: line 154 to 157

File: ako-gateway-api/nodes/dequeue_ingestion.go [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+		parentNs, _, parentName := lib.ExtractTypeNameNamespace(gatewayNsName)
+		modelName := lib.GetModelName(lib.GetTenant(), akogatewayapilib.GetGatewayParentName(parentNs, parentName))
+
+		modelFound, modelIntf := objects.SharedAviGraphLister().Get(modelName)
+		if !modelFound || modelIntf == nil {
+			utils.AviLog.Errorf("key: %s, msg: no model found: %s", key, modelName)
+			continue
+		}
+		if objType == utils.Secret {
+			handleGateway(namespace, parentName, fullsync, lib.Gateway+"/"+gatewayNsName)
```
[the remark this answers is not in the document]
Comment (pkoshtavmware) [reply ako-gateway-api/nodes/dequeue_ingestion.go]: Changes Done

**Comment by pkoshtavmware** [comment]
Running tool: /usr/local/go/bin/go test -timeout 30s -run ^TestSecretCreateDelete$ github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests/graphlayer

ok  	github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests/graphlayer	10.518s

**Pull request closed without merging**